package.json

```diff
@@ -95,6 +95,7 @@
     "./curriculum/challenges/**/*.md": "node ./tools/scripts/lint/index.js"
   },
   "dependencies": {
+    "bowser": "^2.11.0",
```

Comment:
Client specific dependencies should go inside the client package.json. This *works*, but it breaks the modularity.

Reply:
Thanks for the feedback, really appreciate it. I am fairly new to open source code. Trying to contribute more :).

Reply:
You're very welcome :) Thanks for putting this together.